kickoff/test/base.py

```diff
@@ -155,6 +155,23 @@ def setUp(self):
             db.session.add(r)
 
             r = FirefoxRelease(partials='0,1', promptWaitTime=5,
+                               submitter='joe', version='3.0b3', buildNumber=1,
+                               branch='a', mozillaRevision='defau',
+                               l10nChangesets='ja zu', dashboardCheck=True,
+                               mozillaRelbranch='FOO',
+                               submittedAt=datetime(2005, 2, 3, 4, 5, 6, 7),
+                               shippedAt=datetime(2005, 2, 3, 4, 5, 6, 7),
+                               comment="yet an other amazying comment",
+                               isSecurityDriven=True,
+                               description="Regular new beta released",
+                               enUSPlatforms=None)
+            r.complete = True
+            r.ready = True
+            # Shipped
+            r.status = "postrelease"
```

Comment:
We switched to using "shipped" instead of "postrelease". Another candidate to move to constants. :)